ext/reflection/php_reflection.c

```diff
@@ -878,7 +884,10 @@ static zval *property_get_default(zend_property_info *prop_info) {
 		ZVAL_DEINDIRECT(prop);
 		return prop;
 	} else {
-		return &ce->default_properties_table[OBJ_PROP_TO_NUM(prop_info->offset)];
+		if (UNEXPECTED(zend_update_class_constants(ce) != SUCCESS)) {
+			return &EG(uninitialized_zval);
+		}
+		return &CE_DEFAULT_PROPERTIES_TABLE(ce)[OBJ_PROP_TO_NUM(prop_info->offset)];
```

Comment:
Note though that this is actually kind of breaking. We're changing the default value to show the evaluated constant, rather than the constant AST. This was already the case when not using opcache, because we're evaluating the AST in-place. However, when using opcache, the AST is locked in shm and copied into "userspace" to be evaluated there. Adjusting this makes sense IMO to match the behavior of opcache and non-opcache, but I think that's better fixed on `master`.

Maybe you can propose this in a separate PR on `master`, and adjust tests to disable opcache (`opcache.enable_cli=0` in `--INI--`).